lambdas/process-journey-event/src/main/java/uk/gov/di/ipv/core/processjourneyevent/statemachine/stepresponses/CriStepResponse.java

```diff
@@ -17,7 +21,20 @@ public class CriStepResponse implements StepResponse {
 
     private String criId;
 
-    public Map<String, Object> value() {
-        return Map.of("journey", String.format(CRI_JOURNEY_TEMPLATE, criId));
+    private String context;
+
+    private String scope;
+
+    public Map<String, Object> value() throws URISyntaxException {
```

Comment:
If you want a checked exception, then I think we should wrap in our own type - perhaps reusing `JourneyEngineException` - it feels a bit weird to expose this in the interface signature.

Alternatively, I'd be happy to catch this and re-throw as a runtime exception if we're happy that this will never happen in any reasonable scenario.

Reply:
I think it's never supposed to happen because we would catch this syntax error in any e2e test touching the incorrect-format cri state. I'd like to go with a Runtime exception I think